### PR TITLE
Web support enhancement

### PR DESCRIPTION
**Pull request by Sami32**
This change is [Reviewable](https://reviewable.io/reviews/digitalmediaserver/digitalmediaserver/119)

### CODE REVIEW

File: src/main/java/net/pms/configuration/WebRender.java [hunk 1]
```diff
@@ -129,7 +130,7 @@ public boolean load(File f) {
 		configuration.addProperty(SUPPORTED, "f:jpg m:image/jpeg");
 		configuration.addProperty(SUPPORTED, "f:png m:image/png");
 		configuration.addProperty(SUPPORTED, "f:gif m:image/gif");
-		configuration.addProperty(SUPPORTED, "f:tiff m:image/tiff");
+//		configuration.addProperty(SUPPORTED, "f:tiff m:image/tiff");
```
Comment (Sami32) [inline src/main/java/net/pms/configuration/WebRender.java]: Should i need to comment it, or the below TIFF limitation is suffisant to avoid issue?

Comment (Nadahar) [reply src/main/java/net/pms/configuration/WebRender.java]: It doesn't matter, these aren't used anyway (no "supported" lines for images remember?).

Comment (Sami32) [reply src/main/java/net/pms/configuration/WebRender.java]: Memory flaw ;)

So shouldn't be better if i clean these supported lines?

Comment (Nadahar) [reply src/main/java/net/pms/configuration/WebRender.java]: The whole class (end the whole web interface stuff) needs a proper refactoring, so I don't care so much about the details either way.

Comment (Nadahar) [reply src/main/java/net/pms/configuration/WebRender.java]: But yeah, ideally, all the images could be removed.

Comment (Nadahar) [reply src/main/java/net/pms/configuration/WebRender.java]: If you change it, remember that I have rebased the branch so you must update your local first.

**Pull request merged**